src/main/java/eunAI/EunAI.java

```diff
@@ -0,0 +1,61 @@
+package eunAI;
+
+import java.util.Scanner;
+
+import eunAI.command.Command;
+import eunAI.command.CommandParser;
+import eunAI.ui.Ui;
+
+
+/**
+ * Represents the main application for managing tasks.
+ * Initializes the UI, storage, and task list, and handles user input in a command loop.
+ */
+public class EunAI {
+
+    private final Storage storage;
+    private final TaskList taskList;
+    private final Ui ui;
+
+    /**
+     * Contructs an EunAI chatbot instance with the specified filePath for task storage.
```

Comment:
```suggestion
     * Constructs an EunAI chatbot instance with the specified filePath for task storage.
```
there's a type on the word 'constructs'